experimental/decorators.md

```diff
@@ -9,18 +9,36 @@ interface Decorator <: Node {
 }
 ```
 
-## Used
+## AccessorProperty
+```js
+interface AccessorProperty <: Node {
```

Comment:
On AccessorProperty, an alternative design is to add `accessor: boolean` to `PropertyDefinition`. Babel introduced a new type so that we can support https://github.com/tc39/proposal-grouped-and-auto-accessors based on the new AST node type. See also discussions [here](https://github.com/babel/babel/pull/13681).

Reply:
I think a new node type makes sense. 👍